valheim_plus.cfg

```diff
@@ -343,6 +346,12 @@ baseUnarmedDamage=120
 ; When changed to true, you will not be permitted to place a crop within the grow radius of another crop
 cropNotifier=false
 
+; How many seconds each comfort level contributes to the rested bonus, in percentage.
+restSecondsPerComfortLevel=0
+
+; Change the death penalty in percentage - set to -100 to disable it.
```

Comment:
For clarity: "set to -100 to disable the skill death penalty entirely" also would yo mind adding an example to how this is used? Like "setting the value to 10 would lower all skills by 10% on player death" or something.

Reply:
It's actually reverse, so setting it to 50 would increase death penalties by 50%, and setting it to -50 would decrease death penalties by 50%... I can add examples or change how it works, what do you think?

Reply:
No need to change functionality, examples would be great!